app/policies/kondo_policy.rb

```diff
@@ -0,0 +1,10 @@
+class KondoPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+    def index?
+      true
+  end
```

Comment:
Very minor but better correct the indentation?

Reply:
I viewed the file and the indentations seem to be alright.
<img width="235" alt="Screen Shot 2022-01-27 at 0 33 34" src="https://user-images.githubusercontent.com/18746187/151194799-05bd6747-4308-44ab-8acc-d0a7a88e311c.png">

I was so paranoid I copy-pasted in vs code and the indentations seems right.

Reply:
Oh that’s strange! It must be my browser doing something weird! I resolve then :)